conu/backend/docker/image.py

```diff
@@ -418,6 +418,30 @@ def get_volume_options(volumes):
             result += ["-v", str(v)]
         return result
 
+    @classmethod
+    def build(cls, path, tag=None, dockerfile=None):
+        """
+        Build the image from the provided dockerfile in path
+
+        :param path : str, path to the directory containing the Dockerfile
+        :param tag: str, A tag to add to the final image
+        :param dockerfile: str, path within the build context to the Dockerfile
+        :return: instance of DockerImage
+        """
+        if not path:
+            raise ConuException('Please specify path to the directory containing the Dockerfile')
+        client = get_client()
+        response = [line for line in client.build(path,
+                                                  rm=True, tag=tag,
+                                                  dockerfile=dockerfile,
+                                                  quiet=True)]
+        if not len(response):
+            raise ConuException('Failed to get ID of image')
+
+        # get ID from output
+        # b'{"stream":"sha256:39c7bac4e2da37983203df4fcf612a02de9e6f6456a7f3434d1fccbc9ad639a5\\n"}\r\n'
+        image_id = response[0].decode('utf-8')[11:-6]
```

Comment:
this looks pretty fragile, no idea whether we should add some error handling in it